src/app/portlets/content-types/form/content-types-form.component.html

```diff
@@ -26,55 +26,55 @@
         <div class="ui-g">
             <div class="ui-g-6">
                 <div class="form__group" *ngIf="isEditMode">
-                    <b>{{i18nMessages['Variable']}}:</b>
+                    <b>{{i18nMessages['contenttypes.variable']}}:</b>
                     <a href="#">{{data.variable }}</a>
                 </div>
                 <div class="form__group">
-                    <label for="content-type-form-description" class="form__label">{{i18nMessages['description']}}</label>
+                    <label for="content-type-form-description" class="form__label">{{i18nMessages['contenttypes.form.label.description']}}</label>
                     <input pInputText type="text" id="content-type-form-description" name="description" formControlName="description" />
                 </div>
                 <div class="form__group">
-                    <label for="content-type-form-host" class="form__label">{{i18nMessages['Host-Folder']}}</label>
+                    <label for="content-type-form-host" class="form__label">{{i18nMessages['contenttypes.form.field.host.folder']}}</label>
                     <site-selector-component id="content-type-form-host" formControlName="host" [system]="true"></site-selector-component> 
                 </div>
                 <div class="form__group">
-                    <label for="content-type-form-workflow" class="form__label">{{i18nMessages['Workflow']}}</label>
+                    <label for="content-type-form-workflow" class="form__label">{{i18nMessages['contenttypes.form.label.workflow']}}</label>
                     <p-dropdown [options]="workflowOptions" [style]="{'min-width':'90px'}" id="content-type-form-workflow" name="workflow" formControlName="workflow"></p-dropdown>
                 </div>
-                <span id="field-workflow-hint" class="form__group-hint" *ngIf="form.get('workflow').disabled">{{i18nMessages['Only-Default-Scheme-is-available-in-Community']}}</span>
+                <span id="field-workflow-hint" class="form__group-hint" *ngIf="form.get('workflow').disabled">{{i18nMessages['contenttypes.form.hint.only.default.scheme.available.in.Community']}}</span>
```

Comment:
is this a error message?, if it is so better: 

contenttypes.form.hint.error.only.default.scheme.available.in.Community